Explainer/PresentationParser.py

```diff
@@ -0,0 +1,33 @@
+from pptx import Presentation
+
+
+class PresentationParser:
```

Comment:
Nice and useful class, which is well separated from the rest of the code for future use